boards/arm/rpi_pico/rpi_pico-pinctrl.dtsi

```diff
@@ -15,4 +15,22 @@
 			input-enable;
 		};
 	};
+
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <I2C0_SDA_P4>, <I2C0_SCL_P5>;
+			bias-pull-up;
```

Comment:
I'm not sure pull up should be enabled by default, as it is common to have external pull ups (for example, if I2C is used for a sensor that already has pull ups)

Reply:
agreed, ref https://github.com/zephyrproject-rtos/zephyr/pull/42513#issuecomment-1036411360

Reply:
Yes, you are right, I removed it.